### ISSUE TITLE
can't open files (strchars not found)

### ISSUE THREAD

**Issue by klauern** [scrooloose/nerdtree#517]
I just recently updated NERDTree and now I get this error whenever trying to do anything in NERDTree:

```
Error detected while processing function nerdtree#ui_glue#invokeKeyMap..73..114..231..238:
line    3:
E117: Unknown function: strchars
E15: Invalid expression: strchars(a:line[0:numLeadBytes-1])
```

I think it has to do with a commit made a week ago [displayed here](https://github.com/scrooloose/nerdtree/commit/677a83b2b63fda1c52205acd39973fe1b44e8b54#diff-f094542718c161e8482e397da4e1f499R292)

I'm not on the newest Vim (7.2.411), but I'm kind of stuck at the moment on Oracle Linux 6.

**Comment by yous** [comment]
`strchars()` is added in Vim 7.3: http://vimdoc.sourceforge.net/htmldoc/version7.html#added-7.3. I found that ingo-library defines compatible function [`ingo#compat#strchars()`](https://github.com/vim-scripts/ingo-library/blob/master/autoload/ingo/compat.vim):

``` vim
if exists('*strchars')
    function! ingo#compat#strchars( expr )
        return strchars(a:expr)
    endfunction
else
    function! ingo#compat#strchars( expr )
        return len(split(a:expr, '\zs'))
    endfunction
endif
```

But I'm not sure this is reliable.

**Comment by PhilRunninger** [comment]
This is fixed in commit [4428aba](https://github.com/scrooloose/nerdtree/commit/4428aba2a299cbd117c80e7d18068b2cbb222fff) using @yous 's posted workaround above.
